OPS.md

```
@@ -0,0 +1,2 @@
+- [ ] 2026-09-06 · prod · Configure the SDK publisher GitHub App, npm trust, and tag rules before the first release request — `SDK_APP_ID`, `SDK_APP_PRIVATE_KEY`; protect `v*`; require `Release contract and tooling`. See [rollout](docs/sdk-releases.md#recovery-and-rollout).
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)